components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java

```diff
@@ -143,6 +147,9 @@ public class APIAdminImpl implements APIAdmin {
     private static final Log log = LogFactory.getLog(APIAdminImpl.class);
     protected ApiMgtDAO apiMgtDAO;
     protected LabelsDAO labelsDAO;
+    private CertificateMgtUtils certificateMgtUtils = CertificateMgtUtils.getInstance();
+    private static final String MUTUAL_TLS = "Mutual-TLS";
```

Comment:
use the existing constants class

Reply:
no any pre declared constant for this. usage reside only within the class